docs/Pipfile

```diff
@@ -8,5 +8,6 @@ Sphinx = "~=2.3"
 sphinx_rtd_theme = "===0.4.2"
 # just for http.server py2/py3 syntax consistency in manual doc testing
 future = "~= 0.16"
-runway = {editable = true, path = "./.."}
+runway = {editable = true,path = "./.."}
 sphinxcontrib-apidoc = "*"
+jsx-lexer = "*"
```

Comment:
`docs/requirements.txt` needs to be updated with this change.

run `pipenv lock --requirements > requirements.txt` from `docs/`. you will then need to manually remove the entry for `runway`. iirc it gets added to the top of the file as a relative path (e.g. `../.`)